client/src/flowTypes.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 export type Restaurant = {|
-  id: number,
+  id?: number,
```

Comment:
mmm.. can't see why you made the `id` an optional property. can you pls elaborate?

Reply:
When adding a new restaurant, we don't set the id so it's not presented. However, when we fetch restaurants from the server (or read the response for adding a restaurant, for that matter) we get the id from the server.